cvrf2csaf/section_handlers/document_notes.py

```diff
@@ -3,11 +3,52 @@
 
 
 class DocumentNotes(SectionHandler):
-    def __init__(self):
-        super().__init__()
+    def __init__(self, config):
+        super().__init__(config=config)
+
+        self.enum_categories = {
+            'description',
+            'details',
+            'faq',
+            'general',
+            'legal_disclaimer',
+            'other',
+            'summary'
+        }
+
+    def _process_mandatory_and_optional(self, root_element):
+        notes = []
+
+        for elem_note in root_element.Note:
+
+            # mandatory
+            new_note = dict(
+                text=elem_note.text,
+                category=elem_note.get('Type'),
+            )
+
+            if new_note['category'].lower() not in self.enum_categories:
```

Comment:
This will most likely fail on the valid CVRF input of `Legal Disclaimer` which would be converted to `legal disclaimer`. But the valid CSAF value is `legal_disclaimer`.

Reply:
Replace whitespace with underscore -> will be fixed above

Reply:
IMO this code section (line 30-37) can be removed, since this is validated during xsd validation (the Type has an Enum of values, which are validated on the input). So the if branch is never executed

https://github.com/csaf-tools/CVRF-CSAF-Converter/blob/10e9ef8259d55b835704bc695067cb03539b143a/schemata/cvrf/1.2/cvrf.xsd#L292

Reply:
Fixed in a5658c2a7d45969fe6243bdbb3dba2c45cfec816